src/pages/_document.js

```diff
@@ -23,7 +23,7 @@ export default class MyDocument extends Document {
     // If we're deployed to anywhere other than the server root, we'll have to
     // store our root path that here so that the client can access it.
     const script = {
-      __html: `window.BASE_URL = '${baseUrl}';`,
+      __html: `window.BASE_URL = '${baseUrl}'; window.IS_DEV = '${isDev}';`,
```

Comment:
Glad I tested this! We need the literal value `false` here, not the string `"false"`. You should just be able to template `${isDev}` instead of `'${isDev}'`.